modules/ROOT/pages/05-eventing/eventing-src-sub.adoc

```diff
@@ -7,18 +7,26 @@ include::_attributes.adoc[]
 [#eventing-channel]
 == Channel(Sink)
 
-The https://en.wikipedia.org/wiki/Event-driven_architecture#Event_channel[channel or sink] is an interface between the <<eventing-source,event source>> and the <<eventing-subscriber,subscriber>>.The channels are built in to store the incoming events and distribute the event data to the subscribers. When forwarding event to subscribers the channel transforms the event data as per http://cloudevents.io[CloudEvent] specification.
+The https://en.wikipedia.org/wiki/Event-driven_architecture#Event_channel[channel or sink] is an interface between the <<eventing-source,event source>> and the <<eventing-subscriber,subscriber>>. The channels are built in to store the incoming events and distribute the event data to the subscribers. When forwarding event to subscribers the channel transforms the event data as per http://cloudevents.io[CloudEvent] specification.
 
 [#eventing-create-event-channel]
 === Create Event Channel
 .link:{github-repo}/{eventing-repo}/knative/channel.yaml[channel.yaml]
 [source,yaml,linenums]
 ----
-include::ROOT:example$05-eventing/channel.yaml[]
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Channel
+metadata:
+  name: ch-event-greeter #<1>
```

Comment:
I've included the whole snippet so that we can use ordered lists like `<1>` and later reference these numbers in the legend. I noticed we do that in some documents but not here.